src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md

```diff
@@ -25,28 +28,36 @@ capture and transmit the event.
 
 : _Optional_. A list of packages that were installed as part of this SDK or the
   activated integrations. Each package consists of a `name` in the format
-  `source:identifier` and a semver `version`. If the source is a git repository,
-  a checkout link and git reference (branch, tag or SHA) should be used.
+  `source:identifier` and `version`.  
```

Comment:
I noticed the Go SDK doesn't follow this format.

What are the valid values for `source`? Is it free-form? Is it used for anything?

Reply:
From ingestion perspective, this is completely freeform, but I'm guessing that we might need some recommendation here.